options/locale/locale_zh-CN.ini

```diff
@@ -1983,6 +1983,10 @@ topic.done=保存
 topic.count_prompt=您最多选择25个主题
 topic.format_prompt=主题必须以字母或数字开头，可以包含连字符 (-)，并且长度不得超过35个字符
 
+container = 容器仓库
```

Comment:
Don't change location files except en-US. We use crowdin for translations.

Reply:
remove in new commit